cmd/account/verify-secrets.go

```diff
@@ -19,52 +19,54 @@ import (
 )
 
 const (
-	checkSecretsUsage = "The check-secrets command should have only 0 or 1 arguments"
+	verifySecretsUsage = "The verify-secrets command should have only 0 or 1 arguments"
 )
 
-// newCmdCheckSecrets implements the check-secrets command
-// which checks AWS credentials managed by AWS Account Operator
-func newCmdCheckSecrets(streams genericclioptions.IOStreams, flags *genericclioptions.ConfigFlags) *cobra.Command {
-	ops := newCheckSecretsOptions(streams, flags)
-	checkSecretsCmd := &cobra.Command{
-		Use:               "check-secrets [<account name>]",
-		Short:             "Check AWS Account CR IAM User credentials",
+// newCmdVerifySecrets implements the verify-secrets command
+// which verifies AWS credentials managed by AWS Account Operator
+func newCmdVerifySecrets(streams genericclioptions.IOStreams, flags *genericclioptions.ConfigFlags) *cobra.Command {
+	ops := newVerifySecretsOptions(streams, flags)
+	verifySecretsCmd := &cobra.Command{
+		Use:               "verify-secrets [<account name>]",
+		Short:             "Verify AWS Account CR IAM User credentials",
 		DisableAutoGenTag: true,
 		Run: func(cmd *cobra.Command, args []string) {
 			cmdutil.CheckErr(ops.complete(cmd, args))
 			cmdutil.CheckErr(ops.run())
 		},
 	}
 
-	checkSecretsCmd.Flags().StringVar(&ops.accountNamespace, "account-namespace", common.AWSAccountNamespace,
+	verifySecretsCmd.Flags().StringVar(&ops.accountNamespace, "account-namespace", common.AWSAccountNamespace,
 		"The namespace to keep AWS accounts. The default value is aws-account-operator.")
-	checkSecretsCmd.Flags().BoolVarP(&ops.verbose, "verbose", "v", false, "Verbose output")
+	verifySecretsCmd.Flags().BoolVarP(&ops.verbose, "verbose", "v", false, "Verbose output")
+	verifySecretsCmd.Flags().BoolVarP(&ops.all, "all", "a", false, "Verify all Account CRs")
```

Comment:
For UX, can we make `all` be `-A`?  `-a` is used by other commands to pass the account-name.

Reply:
`-A` matches the `oc` command in v4 as well.

Reply:
Done